.devcontainer/docker-compose.yml

```diff
@@ -86,6 +86,14 @@ services:
       - database-root
       - database-tenant
 
+  mailtrap:
+    image: mailhog/mailhog
+    container_name: mailtrap-pool
+    hostname: mailtrap-pool
+    ports:
+      - "1026:1025"
+      - "8026:8025"
+
```

Comment:
can mailtrap server alrady be used?

Reply:
for development only, the "hostname" for sending emails is "mailtrap-pool" (check seeds)